packages/cascara/src/modules/ActionEdit/ActionEdit.js

```diff
@@ -35,9 +35,9 @@ const ActionEdit = ({ dataTestIDs, editLabel = 'Edit' }) => {
   /**
    * this seems like ugly, we need to find a better way
    * to ease testing.. */
-  let cancelTestId = {};
-  let editTestId = {};
-  let saveTestId = {};
+  const cancelTestId = {};
+  const editTestId = {};
+  const saveTestId = {};
```

Comment:
@lordpixel I noticed these during this fix and created a task for removing these test id things here in addition to removing the props which we already did. <3

Reply:
I added a comment on that task to explain why we need them. We'll need another way of specifying these.